### ISSUE TITLE
Puzzle

### ISSUE THREAD

**Issue by jimvnt** [vncsms/Metin2FishBot#25]
On the saturday the 14th of october they fixed the puzzlebot. It doesnt work anymore. Please bring an update and more new features. Thanks mate

**Comment by vncsms** [comment]
They? They changed something? Which server?

**Comment by jimvnt** [comment]
> They? They changed something? Which server?

Yes. On last saturday after the maintenance the puzzle bot is not working.

Fishingbot is working but the puzzle bot no more.

Emerald Server

**Comment by jimvnt** [comment]
> They? They changed something? Which server?

Did you find a solution? And my other question is: Is the bot detectable by the systems of gameforge that I am using an bot or something? Or can I get banned? Can they now that I'm using this script or something.

**Comment by vncsms** [comment]
Not yet, in my country I can´t play metin2 from GameForge, just OnGame. It will be hard for me to detect this problem.
Until now, I haven´t heard of any dection from Gameforge. There was one people who said that one server was temporally banning people who are fishing for too long, and because they are too good on fishing.

The game doesn´t know who are using the bot, the bot don´t inject anything on the game, just take pictures and use the mouse and keyboard. But they can try to detect who are good at fishing.

I can´t guarantee that you are not going to be banned. Since my code is open, Gameforge knows that my bot exists, so they are probally working in something.
